scripts/partitioned/migration/1-create-tables.sql

```diff
@@ -0,0 +1,40 @@
+DROP TABLE IF EXISTS public.journal_partitioned CASCADE;
```

Comment:
IMHO script should fail in case of any conflicts and require some "manual" decisions to be done.

Reply:
I've removed all 'DROP' and 'IF NOT EXISTS' statements.